low-level/sensors/impl/generic/node_hw.py

```diff
@@ -480,6 +481,58 @@ def run(self):
             logger.warning(f"{self.SENSOR_NAME} Node hw monitoring disabled")
             self.shutdown()
 
+    def _check_missmatch_resource_status(self):
+        for sensor_id in self.faulty_resources:
+            if self.faulty_resources[sensor_id]['resource'] in self.fru_types.keys():
+                if self.faulty_resources[sensor_id]['resource'] in [
+                    self.TYPE_PSU_SUPPLY, self.TYPE_PSU_UNIT]:
+                    sdr_get_data, _ = self._get_sensor_sdr_props(sensor_id)
+                    if sdr_get_data and 'States Asserted' in sdr_get_data:
+                        #  'States Asserted': 'Power Supply, Presence detected'
+                        resource_state = re.sub(',  +', ', ', re.sub('[\[\]]','',
+                            sdr_get_data['States Asserted']).replace('\n',','))
+                        sensor_status = resource_state.split(',')
+                        sensor_status = sensor_status[0] if len(sensor_status) == 1 \
+                            else sensor_status[1]
+                        sensor_status = sensor_status.strip()
+                        if sensor_status in ["", "Presence detected"]:
+                            self._create_sel_event_entry(sensor_id)
+                else:
+                    _, sensor_get_data, _ = self._get_sensor_props(sensor_id)
+                    if sensor_get_data and 'Assertion Events' in sensor_get_data \
+                            and 'Status' in sensor_get_data:
+                        # 'Status': 'ok'
+                        # 'Assertion Events': ''
+                        if sensor_get_data['Assertion Events'] in [""] \
+                            and sensor_get_data['Status'] == 'ok':
+                            self._create_sel_event_entry(sensor_id)
+
+    def _create_sel_event_entry(self, sensor_id):
```

Comment:
Dont need this function, we are not going to introduce any SEL entry ourselves

Reply:
Yes. deleted.
Done